### PR TITLE
Fix tflint issues in base workspace

### PR DESCRIPTION
**Pull request by tamirkamara**
## What is being addressed

Following #2287, this fixes terraform issues in the base workspace.

### CODE REVIEW

File: templates/workspaces/base/terraform/providers.tf [hunk 1]
```diff
@@ -8,9 +8,9 @@ terraform {
       source  = "hashicorp/azuread"
       version = "=2.20.0"
     }
-    random = {
-      source  = "hashicorp/random"
-      version = "=3.1.2"
+    null = {
```
Comment (guybartal) [inline templates/workspaces/base/terraform/providers.tf:11]: Not sure about this change, is that came from the linter?

Comment (tamirkamara) [reply templates/workspaces/base/terraform/providers.tf:15]: Thanks. switched my mistake instead of adding.

Comment (tamirkamara) [reply templates/workspaces/base/terraform/providers.tf:11]: Actually, it wasn't by mistake. The updated version is correct.

Comment (guybartal) [reply templates/workspaces/base/terraform/providers.tf:11]: Ok, thanks

**Review by guybartal — APPROVED** [review]
LGTM

**Comment by tamirkamara** [comment]
/test-force-approve
tests were successful [here ](https://myexpense.operations.dynamics.com/?cmp=1604&mi=DefaultDashboard)and recent merge has no impact on this PR.

**Pull request merged**